content/en/guides/os-commercial-ecosystem.md

```diff
@@ -0,0 +1,175 @@
+
+* **Community management:** The community is often the strongest asset a successful open source project can have, and investing resources in growing a healthy community is a best practice. Often, an enthusiastic community grows from a beloved, free edition of a project, and therefore it is a mistake to invest only in community growth for a commercial edition.
+
+* **Access to test infrastructure/environments:** Some open source applications require dedicated servers, specialist tools, automated quality control and/or security hardening. Dedicating resources to a paid and supported edition while starving resources for a free, community edition is a poor strategy, because free community editions often facilitate the first steps that potential customers will take. Ensure open source versions and any open source developers have access to the same tools, checks and automation that your staff developers or paid versions are run on. Parity across different versions and collaborators is key.
+
+
+## 3. Track progress and make adjustments
+
+Of course, your effort to create a healthy, commercial ecosystem around open source is a moving target. You must keep the people overseeing your governance structure involved, and keep your community providing feedback. “Listen, measure and adjust,” is good advice on this front. The members of your community can provide extraordinarily good feedback on how to make needed improvements and reach more people. Remember that each community is different.
+
+*Find metrics for each community that you're working with. I tend to find metrics based on what a specific community is feeling as pain, and try to change those metrics for the better. There is no one magic assessment where if you look at six metrics, ranging from pull requests to contributor numbers, you can suddenly pronounce your community and ecosystem healthy. You could have a very small project that is extraordinarily healthy because it has half a dozen core contributors and a dozen people who are active but not maintainers. There might be healthy discussions, and pull requests might be handled in a speedy manner, and that might be an incredibly healthy community even though it's not going to have a million stars or forks on GitHub."* – Sarah Novotny, Open Source Wonk, Azure Office of the CTO
+
+The bottom line is that there is much more than code to adjust as you maintain your healthy open source ecosystem. Focus on outreach through training, events, and funding, putting your community first. Here are some important tenets to keep in mind:
+
+**Invest wisely in education.**
+
+* Training – Education and training offerings serve a virtuous cycle, where trained users advance the cause of your open source inventions.
```

Comment:
Should training, certifications, documentation, etc... be in section 2 on resources?

Reply:
yes, they can be two other resources to look at. I'll change it